lib/versioned/test.js

```diff
@@ -80,6 +80,7 @@ function Test(directory, pkgVersions, opts = {}) {
   this.duration = 0
   this.allPkgs = !!allPkgs
   this.strict = !!opts.strict
+  this.type = pkg.type
```

Comment:
we have unit test for this file, might be good to add some assertions around type.